spec/services/processors/external/pull_requests_spec.rb

```diff
@@ -22,4 +22,18 @@
         .to change { ExternalPullRequest.count }.by(1)
     end
   end
+
+  context 'when user has an open pull request in a given project which gets merged' do
+    let!(:external_pull_request) do
+      ExternalPullRequest.destroy_all
```

Comment:
Why are we doing this?

Reply:
My mistake I was trying something here. Will delete this as it's not necessary.